extra/gemm/hip_matmul.py

```diff
@@ -93,22 +111,32 @@
       }}
     }}
   }}
-}}""")
+}}"""
 
-prog = HIPProgram(device.device, "test", lib)
+if DEBUG > 1: print(prog_str)
+lib = compile_hip(prog_str)
```

Comment:
use `device.compiler.compile` instead of importing `compile_hip` directly?

Reply:
done!